docs/static/monitoring-apis.asciidoc

```diff
@@ -517,64 +690,29 @@ Logstash.
 
 Example response:
 
-[source,js]
+["source","sh",subs="attributes"]
 --------------------------------------------------
 {
-    "time": "2017-01-12T12:09:45-08:00",
```

Comment:
@jsvd I try to update the examples from time-to-time so they don't look stale (because the date is old or the format of events changes etc). I always replace the entire example instead of just tweaking parts because it's easy to do and less error prone.  Again, sorry if it took you a bit more time to scan through the changes....